web/cypress/integration/morecast-page.spec.ts

```diff
@@ -101,134 +107,144 @@ describe('MoreCast Page', () => {
         .should('have.css', 'height', '0px')
     })
 
-    it('Temp & RH graph displays svg graphics with toggles', () => {
-      // Check if svg elements are displayed in the graph
-      cy.getByTestId('hourly-observed-temp-symbol').should('have.length', numOfObservations - 1)
-      cy.getByTestId('hourly-observed-temp-path').should('not.have.class', 'hidden')
-      cy.getByTestId('hourly-observed-rh-symbol').should('have.length', numOfObservations)
-      cy.getByTestId('hourly-observed-rh-path').should('not.have.class', 'hidden')
+    it('Temp & RH Graph should be displayed', () => {
+      const checkNumOfTempMarkers = (num: number) => {
+        cy.getByTestId('temp-rh-graph')
+          .find(`.cartesianlayer > .subplot > .plot > .scatterlayer > .trace > .points > .point`)
+          .should('have.length', num)
+      }
+      const checkTempPlume = (shouldThere: boolean) => {
```

Comment:
`const checkTempPlume = (expectedLength: int) => {`

Reply:
It's actually checking the number of svgs being displayed(is there just lines or lines with 2 more lines that make up the plume) which is why I just need the boolean

Reply:
I guess the name is just weird - "shouldThere" - can't think of a better name for you though....

Reply:
I am going with `shouldBeDisplayed`, thoughts?